src/services/groups/index.ts

```diff
@@ -66,13 +67,31 @@ export const fetchGroups = async () => {
 export const getGroupsPage = async (
   count: number,
   startIndex: number = 1,
+  me: boolean = false,
   filter?: string
 ) => {
-  let url = `${BASE_URL}/iam/group/search?count=${count}&startIndex=${startIndex}`;
-  if (filter) {
-    url += `&filter=${filter}`;
+  if (me) {
```

Comment:
`me` is not a boolean but is a `User` object containing the list of groups uuid belonging to me, and it is a parameter passed from the caller
`const my_groups = fetcMyGroups(me)`